core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -89,28 +93,49 @@ public static synchronized TachyonFS get(final TachyonURI tachyonURI) throws IOE
         throw new IOException("Invalid Tachyon URI: " + tachyonURI + ". Use " + Constants.HEADER
             + "host:port/ ," + Constants.HEADER_FT + "host:port/");
       }
-      return new TachyonFS(tachyonURI);
+
+      boolean useZookeeper = scheme.equals(Constants.SCHEME_FT);
+      tachyonConf.set(Constants.USE_ZOOKEEPER, Boolean.toString(useZookeeper));
+      tachyonConf.set(Constants.MASTER_HOSTNAME, tachyonURI.getHost());
+      tachyonConf.set(Constants.MASTER_PORT, Integer.toString(tachyonURI.getPort()));
```

Comment:
That is correct. 

The next step is to actually remove the TachyonConf input argument and let the file based properties file as the main source of truth for customization.
I would love to have the new PR merged to master first so we could add iterative changes and improvements.
It is super hard to maintain this branch with new developments coming in.